Translation Script/Map133.csv

```diff
@@ -191,37 +194,33 @@ AlstoトMeriaの花言葉は『未来への憧れ』だ。
 Did those guys really...?"
 "ロメ/アル","ロメ/アル"
 "\n<ロメリア>ごめんね。
-兄貴までこんなことになって・・・","\n<ロメリア>ごめんね。
-兄貴までこんなことになって・・・"
+兄貴までこんなことになって・・・","\n<Meria>Sorry.
+Even my brother is in this predicament..."
 "気にするな","I was worried."
 "お前のせいだぞ","This is your fault."
 "\n<ロメリア>だって三人組だって思わなかったんだもん。
-淫魔の癖に。","\n<ロメリア>だって三人組だって思わなかったんだもん。
-淫魔の癖に。"
+淫魔の癖に。","\n<Meria>I didn't think there would be 3 of them
+That's a bit odd for succubi."
 "\n<ロメリア>ぐぅぅ・・・
-あっ。お腹が・・・","\n<ロメリア>ぐぅぅ・・・
-あっ。お腹が・・・"
+あっ。お腹が・・・","\n<Meria>Guhhhh...
+Ahh, my stomach..."
 "\n<ロメリア>こんな時に悪いんだけど
 何か食べるものあったら持ってきてくれない？
-お腹ぺこぺこ。","\n<ロメリア>こんな時に悪いんだけど
-何か食べるものあったら持ってきてくれない？
-お腹ぺこぺこ。"
+お腹ぺこぺこ。","\n<Meria>I know now's a bad time, but can you bring me
+any food you find? I'm so hungry."
 "\n<アルス>すまん。
 まさか相手が3人組だとは・・・
-ミイラ取りがミイラになったな。","\n<アルス>すまん。
-まさか相手が3人組だとは・・・
-ミイラ取りがミイラになったな。"
+ミイラ取りがミイラになったな。","\n<Alsto>I'm sorry.
+I never would have guessed there'd be 3 of them... \nThe hunters have become the hunted."
 "\n<アルス>もしこの館から逃げられるチャンスがあるなら
 お前ひとりでも迷わず逃げるんだぞ。
-いいな？","\n<アルス>もしこの館から逃げられるチャンスがあるなら
-お前ひとりでも迷わず逃げるんだぞ。
-いいな？"
+いいな？","\n<Alsto>If you get a chance to escape this mansion, don't
+hesitate to take it. Got it?"
 "\n<アルス>あとまぁ、悪いけど
 何か食い物があったら持ってきてくれないか？
-腹が減った・・・","\n<アルス>あとまぁ、悪いけど
-何か食い物があったら持ってきてくれないか？
-腹が減った・・・"
-"梯子","梯子"
+腹が減った・・・","\n<Alsto>Oh, sorry, but if you find any food, could you please 
```

Comment:
Maybe remove the comma between oh and sorry. Having three in a sentence like that might be too much ^^